Service/Order/ComposeRefund.php

```diff
@@ -37,7 +37,6 @@ public function execute(Creditmemo $creditmemo, float $amount, Order $order): ar
             'amount' => min($this->roundAmt($amount) * -1, $grossAmount),
             'currency' => $order->getOrderCurrencyCode(),
             'initiate_payment_to_buyer' => true,
-            'tax_subtotals' => $this->getTaxSubtotals($lineItems),
```

Comment:
Refund request doesn't require tax_subtotal: https://api-docs.two.inc/openapi/checkout-api/#operation/refund_order_handler.post

Reply:
Don't think refund endpoint even cares if it exists or not